metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/variable_save_test.clj

```diff
@@ -95,10 +89,10 @@
           (is (= nil (:revision-id response)))
           (is (= 409 (:status response)))
           (is (= 404 (:status find-response)))
-          (is (= [(format (str "The provider id [%s] and variable name [%s] "
+          (is (= [(format (str "The provider id [%s] and :variable-name [%s] "
```

Comment:
We should back out these changes for consistency with error messages.

Reply:
Done.